ola.spec.in

```diff
@@ -107,21 +103,15 @@ cp debian/ola.udev %buildroot/%{_udevrulesdir}
 %{_libdir}/pkgconfig/*
 
 %files -n python2-%{name}
-%{python2_sitelib}/ola
```

Comment:
we can't do the whole folder because that would include all sub-folders but i would like to have the directories specified whenever possible, because that also specifies the ownership over that folder